action_server/src/sema4ai/action_server/package/_package_build_cli.py

```diff
@@ -275,20 +277,23 @@ def handle_package_command(base_args: ArgumentsNamespace):
 
         # action-server package build --output-dir=<zipfile> --datadir=<directory> <source-directory>:
         try:
-            retcode = build_package(
+            result = build_package(
                 Path(".").absolute(),
                 output_dir=package_build_args.output_dir,
                 datadir=package_build_args.datadir,
                 override=package_build_args.override,
             )
+
+            if package_build_args.json:
+                print(json.dumps({"package_path": result.package_path}))
         except ActionServerValidationError as e:
             log.critical(
                 bold_red(
                     f"\nUnable to build action package. Please fix the error below and retry.\n{e}",
                 )
             )
-            retcode = 1
-        return retcode
+            return 1
+        return result.return_code
```

Comment:
The function might need some clear return types.

Reply:
Would be nice, the whole file basically uses return code 1 for all errors, so I wont do it here. It would be a bigger refactoring.